xdsl/dialects/gpu.py

```diff
@@ -779,6 +795,7 @@ def verify_(self) -> None:
         SubgroupSizeOp,
         TerminatorOp,
         ThreadIdOp,
+        WaitOp,
         YieldOp,
     ],
     [
```

Comment:
```suggestion
    [
        AsyncTokenType
```
Oops, this is on me, I should have done it before!